EIPS/eip-5732.md

```diff
@@ -0,0 +1,106 @@
+---
+eip: 5732
+title: Simple Commit Interface
```

Comment:
Should you add a reveal function?

Reply:
Nope~ the purpose is all other functions *can* be a reveal function.